tools/wptrunner/wptrunner/browsers/webkitgtk_minibrowser.py

```diff
@@ -0,0 +1,74 @@
+from .base import get_timeout_multiplier   # noqa: F401
+from .webkit import WebKitBrowser
+from ..executors import executor_kwargs as base_executor_kwargs
+from ..executors.executorwebdriver import (WebDriverTestharnessExecutor,  # noqa: F401
+                                           WebDriverRefTestExecutor)  # noqa: F401
+from ..executors.executorwebkit import WebKitDriverWdspecExecutor  # noqa: F401
+
+__wptrunner__ = {"product": "webkitgtk_minibrowser",
+                 "check_args": "check_args",
+                 "browser": "WebKitGTKMiniBrowser",
+                 "browser_kwargs": "browser_kwargs",
+                 "executor": {"testharness": "WebDriverTestharnessExecutor",
+                              "reftest": "WebDriverRefTestExecutor",
+                              "wdspec": "WebKitDriverWdspecExecutor"},
+                 "executor_kwargs": "executor_kwargs",
+                 "env_extras": "env_extras",
+                 "env_options": "env_options",
+                 "run_info_extras": "run_info_extras",
+                 "timeout_multiplier": "get_timeout_multiplier"}
+
+
+def check_args(**kwargs):
+    pass
+
+
+def browser_kwargs(test_type, run_info_data, config, **kwargs):
+    return {"binary": kwargs["binary"],
+            "webdriver_binary": kwargs["webdriver_binary"],
+            "webdriver_args": kwargs.get("webdriver_args")}
+
+
+def capabilities(server_config, **kwargs):
+    args = kwargs.get("binary_args", [])
+    if "--automation" not in args:
+        args.append("--automation")
+
+    return {
+        "browserName": "MiniBrowser",
+        "browserVersion": "2.20",
+        "platformName": "ANY",
+        "webkitgtk:browserOptions": {
+            "binary": kwargs["binary"],
+            "args": args,
+            "certificates": [
```

Comment:
Is this still needed? If it worked I presume https://github.com/web-platform-tests/wpt/pull/19185 wouldn't be needed?

Reply:
Good point. I guess this was also copied from other browser drivers, but it doesn't work for WebKitWebDriver.

Reply:
Mmmm. Looking at the WebKit code it seems it should support it. I will try to check what is wrong here.

Reply:
Ok. This seems to be working as expected. Running the test `content-security-policy/worker-src/service-list.https.sub.html`on my dev machine works both with Epiphany and the WebKitGTK MiniBrowser (no cert error) and the url loads via https. And I don't have that cert installed globaly as opening that very same url in another epiphany window (new process) gives an https certificate error.

However, running the test that appears on the screenshot at https://github.com/web-platform-tests/wpt/pull/18595#issuecomment-531163125 (/cookies/samesite/fetch.https.html) gives me a TLS error if the cert is not installed globally. 

I have hunted this down to be a problem on how wpt tells webkitwebdriver for which domains the certificate is valid. I reported this on #19233